lib/prod-server.js

```diff
@@ -1,5 +1,7 @@
 const { createServer } = require('http')
+const urllib = require('url')
```

Comment:
So the serve scrip trims off the path from the absolute CDN url and uses it as the static origin. I'm not 100% sure if basePath is relevant, but I wanted to preserve its existing behaviour in PS static sites...